remotes/docker/resolver.go

```diff
@@ -313,9 +313,11 @@ func (r *dockerBase) doRequestWithRetries(ctx context.Context, req *http.Request
 	responses = append(responses, resp)
 	req, err = r.retryRequest(ctx, req, responses)
 	if err != nil {
+		resp.Body.Close()
```

Comment:
should `resp` be removed from `responses` on error?

Reply:
no need, the updated responses array goes out of scope

Reply:
oh true, read the `*` in the wrong place.